generator.yaml

```diff
@@ -1,26 +1,54 @@
 resources:
   Queue:
     unpack_attributes_map:
-      fields:
-        DelaySeconds:
-        MaximumMessageSize:
-        MessageRetentionPeriod:
-        KmsMasterKeyId:
-        KmsDataKeyReusePeriodSeconds:
-        Policy:
-        ReceiveMessageWaitTimeSeconds:
-        VisibilityTimeout:
-        FifoQueue:
-        ContentBasedDeduplication:
-        RedrivePolicy:
-        CreatedTimestamp:
-          is_read_only: true
-        LastModifiedTimestamp:
-          is_read_only: true
-        QueueArn:
-          is_read_only: true
+      set_attributes_single_attribute: false
```

Comment:
++ this is cool as a documentation thing, even though `false` is the default value for `set_attributes_single_attribute`